### PR TITLE
txtincDev

### PR DESCRIPTION
**Pull request by txtincorporated**

### CODE REVIEW

**Comment by txtincorporated** [comment]
NOTE that commit 614c86f is the end of work on the modularization tasks for LAB2.

File: lib/server.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+const app = require('./app');
+
+function start() {
+
+  console.log('server starting...');
+
+  http.createServer(app).listen(3000, () => {
+    console.log('Server has started on port 3000.');
+  });
```
Comment (GeoffreyEmerson) [inline lib/server.js]: This whole chunk of code should just be your index.js file in the root. This still functions, of course. It's just just redundant.

**Comment by GeoffreyEmerson** [comment]
Missing tests. :(

**Comment by txtincorporated** [comment]
See PR #61 at https://github.com/codefellows-portland-javascript-401d4/express-single-resource/pull/61 for latest version of LAB1

**Pull request merged**